.github/workflows/tests_scripts.yml

```diff
@@ -6,8 +6,9 @@ on:
       - build_container.sh
       - install_software_layer.sh
       - run_in_compat_layer_env.sh
-      - utils.sh
+      - scripts/utils.sh
       - update_lmod_cache.sh
+      - create_directory_tarballs.sh
```

Comment:
This only makes sure that this workflow is *triggered* when changes are made to `create_directory_tarballs.sh` - we're not actually testing `create_directory_tarballs.sh` directly in this workflow... Could/should we?

Reply:
The test should work now (it did locally). However, the CI will still fail until this PR is merged because the script downloads a tarball of the `main` branch to build the final tarballs and the new subdirectory doesn't exist there yet.